tests/test_helpers.py

```diff
@@ -184,3 +192,101 @@ def test_generate_citation_with_all_fields():
 def test_parse_xml_records_returns_record_iterator():
     records = parse_xml_records("tests/fixtures/datacite/datacite_records.xml")
     assert len(list(records)) == 38
+
+
+def test_parse_date_from_string_success():
+    for date in [
+        "1930",
+        "1930-12",
+        "1930",
+        "1930-12",
+        "1930-12-31",
```

Comment:
Is there a reason for the duplicate date strings in here?

Reply:
A find-and-replace went sideways, fixing it now

Reply:
A find-and-replace went sideways, fixing it now